angularjs-portal-home/src/main/webapp/my-app/marketplace/partials/marketplace-details.html

```diff
@@ -1,120 +1,106 @@
-<div ng-controller="MarketplaceDetailsController as detailsCtrl" class="portlet-frame col-xs-12 col-md-12 portlet-details-page">
-  <div ng-if='loading' class="row details-header no-margin">
-    <loading-gif data-object='obj' data-empty='!loading'></loading-gif>
-  </div>
+<div ng-controller="MarketplaceDetailsController as detailsCtrl" class="portlet-frame portlet-details-page" flex="100">
+    <md-card>
+        <div ng-if='loading' class="row details-header no-margin">
+            <loading-gif data-object='obj' data-empty='!loading'></loading-gif>
+        </div>
+        <div ng-if='error'>
+            <div class="alert alert-danger" role="alert" style='margin: 20px;'>
+                <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
+                <span class="sr-only">Error:</span>
+                {{errorMessage}}
+            </div>
+        </div>
 
-  <div ng-if='error'>
-    <div class="alert alert-danger" role="alert" style='margin: 20px;'>
-      <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
-      <span class="sr-only">Error:</span>
-      {{errorMessage}}
-    </div>
-  </div>
+        <!-- DETAILS HEADER -->
+        <md-card-header tabindex="0" layout="row" class="no-margin" ng-if='!loading && !error'>
+            <md-card-avatar layout="column" layout-align="center center">
+                <portlet-icon class="md-headline"></portlet-icon>
+            </md-card-avatar>
+            <md-card-header-text layout="column" layout-align="center start">
+                <span class="md-headline" hide-xs>{{::portlet.title}}</span>
+                <span class="md-title" hide-gt-xs>{{::portlet.title}}</span>
```

Comment:
1. I used md-healine and md-title just because they're existing CSS classes from Angular Material. It may be better to write our own class, but in the short term I felt it was better to work within the framework and not clutter up the LESS files (since I'm still wrapping my head around them).

2. You would think so, but that change would result in the former span being displayed all the time. I think you'd have to write it like this (but I'm not sure even this would work):

```html
<span class="md-title" hide-gt-xs>{{::portlet.title}}</span>
<span class="md-headline" hide show-gt-xs>{{::portlet.title}}</span>
```